dev/io.openliberty.pages.4.0.internal.factories/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -0,0 +1,116 @@
+cleanup.always_use_parentheses_in_expressions=false
```

Comment:
The copyright template above isn't correct. I just fixed some of these here: https://github.com/OpenLiberty/open-liberty/pull/26714

Reply:
Still looks wrong.